standalone/index.js

```diff
@@ -56,6 +56,7 @@ const customMiddleware = {
       }),
   ],
   VIEW_SETTINGS: [() => window.open('https://wordpress.com/me/notifications')],
+  EDIT_COMMENT: [(store, { editCommentLink }) => window.open(editCommentLink)],
```

Comment:
If we look at `OPEN_POST` and `OPEN_COMMENT` we can see that by passing in the site, post, and comment ids we can better handle this in other middleware. For instance, in Calypso we don't ned to open the window, we just take those ids and form a built-in URL and use `page()` to navigate inside the app to the appropriate spot in comments management.

Does that make sense?

Reply:
To clarify a bit @dmsnell comment, If we dispatch `editComment` with `siteId`, `postId`, `commentId` and `href`, you provide a better context for the middleware to perform side effects (navigation, analytics).

For example, to track this action the standalone middleware needs to `sendMessage` for `masterbar-tracks.js` to catch:

```js
EDIT_COMMENT: [ ( store, { siteId, postId, commentId, href } ) => {
	sendMessage({ action: 'editComment', siteId, postId, commentId });
	window.open(href, '_blank');
} ],
```
and in [Calypso](https://github.com/Automattic/wp-calypso/blob/master/client/notifications/index.jsx):
```js
EDIT_COMMENT: [
	( store, { siteId, postId, commentId. href } ) => {
		this.props.checkToggle();
		this.props.recordTracksEvent( 'calypso_notifications_edit_comment', {
			site_id: siteId,
			post_id: postId,
			comment_id: commentId,
		} );
		page( `/comments/${ siteId }/${ commentId }?action=edit` );
	},
],
```

Reply:
I _almost_ see it now.

So I have to update the notifications panel, Calypso, and a mu-plugin.
Damn it's convoluted to just add a button eh? 😄